src/dash-table/derived/table/fragments.tsx

```diff
@@ -1,6 +1,7 @@
 import * as R from 'ramda';
 import React from 'react';
 import { memoizeOneFactory } from 'core/memoizer';
+import isNil from 'ramda/es/isNil';
```

Comment:
Why not just the one in `import * as R from 'ramda';` above? In fact you use both copies...

Reply:
Weird.. fixed with https://github.com/plotly/dash-table/pull/722/commits/33db6f3df51dc5fa6dcefdaa9752eda7f9ff54d4